sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/util/SearchPagedFlux.java

```diff
@@ -21,7 +20,7 @@
  * element type is {@link SearchResult}, and the page type is {@link SearchPagedResponse}.
  */
 public final class SearchPagedFlux extends PagedFluxBase<SearchResult, SearchPagedResponse> {
-    private final Supplier<Mono<SearchFirstPageResponseWrapper>> metadataSupplier;
+    private final Supplier<Mono<SearchPagedResponse>> metadataSupplier;
```

Comment:
Did you mean to make this change?

Reply:
If so should `SearchFirstPageResponseWrapper` be deleted?

Reply:
No, we still need the wrapper. We cached the first paged instance (in wrapper class) at SearchAsyncClient search method. The first page retriever has set the right first paged instance from top-level. So we don't need to set it here anymore.